pkg/controllers/provisioning/v1alpha1/reallocation/suite_test.go

```diff
@@ -77,6 +72,7 @@ var _ = Describe("Reallocation", func() {
 			},
 			Spec: v1alpha1.ProvisionerSpec{
 				Cluster: &v1alpha1.ClusterSpec{Name: "test-cluster", Endpoint: "http://test-cluster", CABundle: "dGVzdC1jbHVzdGVyCg=="},
+				TTLSeconds: ptr.Int32(300),
```

Comment:
This isn't necessary, right? We default to 300 seconds.

Reply:
Good eye. This is calling out a deeper issue, which is that our defaulting webhooks no longer apply during our tests. In the future, I'd like to wire up knative webhooks for our test suites, but there's an incompatibility of knative/pkg and controller-runtime (both attempt to define --kubeconfig command line argument, resulting in panic). Before we wire up the new webhooks to our test infrastructure, we'd need to migrate to knative controllers or restore controller-runtime webhooks. 

While this is definitely a regression in our e2e testing scope, the only code change that resulted was this one. Now, our validation logic is tested directly in provisioner_validation_test.go. I hope to do full e2e testing by GA (with real EKS clusters) so we will capture the integration point in those tests.

Reply:
I see. Thanks for the in-depth answer!